.github/workflows/ci.yml

```
@@ -37,15 +37,19 @@ jobs:
       matrix:
         runner:
           - ubuntu-latest
-          - macos-latest
+          #- macos-latest
     runs-on: ${{ matrix.runner }}
     steps:
       - uses: actions/checkout@v4
-      - name: Set up JDK
-        uses: actions/setup-java@v4
+      - uses: actions/setup-java@v4
         with:
           java-version: ${{ env.java_version }}
           distribution: ${{ env.java_distribution }}
+      - uses: cvmfs-contrib/github-action-cvmfs@v3
```

v5 promises macOS support: https://github.com/cvmfs-contrib/github-action-cvmfs/releases

have you tested?

```suggestion
      - uses: cvmfs-contrib/github-action-cvmfs@v5
```

---

no luck